docs/src/accelerators.md

```diff
@@ -0,0 +1,98 @@
+
+<img src="assets/momentumcoeffs.png" alt="EKI convergence for different momentum coefficients" width="300"/>    <img src="assets/momentumcoeffs_values.png" alt="Coefficient values" width="300"/>
+
+Below is an example of accelerated ETKI convergence on the same problem, using the `NesterovAccelerator()`.
+
+<img src="assets/etki_momentum.png" alt="ETKI convergence with momentum" width="350"/>
+
+## Background & Implementation
+
+### Nesterov Acceleration for Gradient Descent
+
+In traditional gradient descent, one iteratively solves for $x^*$, the minimizer of a function $f(x)$, by performing the update step 
+
+```math
+x_{k+1} = x_{k} + \alpha  \nabla f(x_{k}), 
+```
+
+where $\alpha$ is a step size parameter.
+In 1983, Nesterov acceleration (also known as Nesterov momentum) was introduced to accelerate gradient descent. In the modified algorithm, the update step becomes 
+
+```math
+x_{k+1} = x_{k} + \beta (x_{k} - x_{k-1}) + \alpha  \nabla f(x_{k} + \beta (x_{k} - x_{k-1})), 
+```
+
+where $\beta$ is a momentum coefficient. Intuitively, the method mimics a ball gaining speed while rolling down a constantly-sloped hill.
+
+### Implementation in EKI Algorithm
+
+Nesterov acceleration can be used to accelerate gradient flows, as shown by [Su et al](https://arxiv.org/abs/1503.01243). EKI, a gradient-free method, can be understood as approximating a form of gradient flow ([Calvello et al](https://arxiv.org/abs/2209.11371)). Additionally, work by [Kovachki and Stuart](https://iopscience.iop.org/article/10.1088/1361-6420/ab1c3a) demonstrated success when using a modified particle-based Nesterov acceleration method. This work inspired the following implementation of accelerators for a variety of EKP processes.
+
+The traditional update step for EKI is as follows, with $j = 1, ..., J$ denoting the ensemble member and $k$ denoting iteration number.
+```math
+u_{k+1}^j = u_{k}^j + C_{k}^{u\mathcal{G}} (\frac{1}{\Delta t}\Gamma + C^{\mathcal{G}\mathcal{G}}_k)^{-1} \left(y - \mathcal{G}(u_k^j)\right)
+```
+
+When using accelerators, this update step is modified to include a term reminiscent of that in Nesterov's acceleration method for gradient descent.
+
+We first compute intermediate values:
+
+```math
+v_k^j = u_k^j+ \beta_k (u_k^j - u_{k-1}^j)
+```
+We then update the ensemble:
+
+```math
+u_{k+1}^j = v_{k}^j + C_{k}^{u\mathcal{G}} (\frac{1}{\Delta t}\Gamma + C^{\mathcal{G}\mathcal{G}}_k)^{-1} \left(y - \mathcal{G}(v_k^j)\right)
+```
+
+The momentum coefficient $\beta_k$ here differs for different accelerators.
+
+- In the ``NesterovAccelerator()``, we recursively compute $\beta_k$ as $\beta_k = \theta_k(\theta_{k-1}^{-1}-1)$ in the  as derived in ([Su et al](https://jmlr.org/papers/v17/15-084.html)). This implementation has generally been found to be the most effective in most test cases.
```

Comment:
These double quotes ` `` ` should be single quotes as above when talking about the code objects otherwise they are formatted with Latex Mathmode and come out italicized rather than "code-like"